src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -95,7 +95,7 @@ private static string GetCliAdbExePath()
 
         #region Functions
 
-        public int TimeToWaitForBootCompletionSeconds { get; set; } = 300;
+        public TimeSpan TimeToWaitForBootCompletionSeconds { get; set; } = TimeSpan.FromMinutes(5);

```

Comment:
Probably want to ditch the `Seconds` suffix?

Reply:
yes, good catch